website/src/docs/uppy.md

```diff
@@ -35,7 +35,7 @@ A site-wide unique ID for the instance.
 If multiple Uppy instances are being used, for example on two different pages, an `id` should be specified.
 This allows Uppy to store information in `localStorage` without colliding with other Uppy instances.
 
-Note that this ID should be persistent across page reloads and navigation—it shouldn't be a random number that's different every time Uppy is loaded.
+Note that this ID should be persistent across page reloads and navigation — it shouldn't be a random number that's different every time Uppy is loaded.
```

Comment:
IIRC em dashes do not require spaces around them, do they?

Reply:
you are right, I did not realise it and changed it for my own readability but a quick google search said that it's typically used without spaces. I'll revert this :smile: